Microsoft.Toolkit.Uwp.SampleApp/Models/Sample.cs

```diff
@@ -169,53 +175,52 @@ public async Task<(string contents, string path)> GetDocumentationAsync()
             {
                 filepath = docMatch.Groups["file"].Value;
                 filename = Path.GetFileName(filepath);
-                localPath = $"ms-appx:///docs/{Path.GetDirectoryName(filepath)}";
+                localPath = $"ms-appx:///docs/{Path.GetDirectoryName(filepath)}/";
             }
 
 #if !DEBUG // use the docs repo in release mode
             string modifiedDocumentationUrl = $"{_docsOnlineRoot}master/docs/{filepath}";
-#else
-            string modifiedDocumentationUrl = DocumentationUrl;
-#endif
 
             _cachedPath = modifiedDocumentationUrl.Replace(filename, string.Empty);
 
+            // Read from Cache if available.
             try
             {
-                using (var request = new HttpRequestMessage(HttpMethod.Get, new Uri(modifiedDocumentationUrl)))
-                {
-                    using (var response = await client.SendAsync(request).ConfigureAwait(false))
-                    {
-                        if (response.IsSuccessStatusCode)
-                        {
-                            var result = await response.Content.ReadAsStringAsync();
-                            _cachedDocumentation = ProcessDocs(result);
-
-                            if (!string.IsNullOrWhiteSpace(_cachedDocumentation))
-                            {
-                                await StorageFileHelper.WriteTextToLocalCacheFileAsync(_cachedDocumentation, filename);
-                            }
-                        }
-                    }
-                }
+                _cachedDocumentation = await StorageFileHelper.ReadTextFromLocalCacheFileAsync(filename);
```

Comment:
Did we want to keep the !DEBUG so we read the latest from dev?

Reply:
The latest is only read in Release mode, a.k.a. in the Store. If it is debug mode, we want to get the local docs, that way we can see changes when we edit the documentation locally.